src/components/AccountWarning/AccountWarning.tsx

```diff
@@ -54,6 +54,25 @@ export function AccountWarning() {
         }
     }, [user, user?.has_active_warning_flag]);
 
+    const mainGoban = (window as any)["global_goban"];
+
+    if (location.pathname.indexOf("game/") > 0) {
+        // On the Game page, we need to delay displaying the warning until the game is over
+        if (liveGameInProgress) {
+            return null;
+        } else if (mainGoban?.engine.phase === "play" && mainGoban.engine.game_id) {
+            setLiveGameInProgress(true);
```

Comment:
Should we check whether it's a correspondence game?

Reply:
Doh, I swear I had the correspondence check, must have lost it along the way.   I guess we could tolerate observers not being warned 😝    But yeah not the intent either.